src/lang/fr.json

```diff
@@ -0,0 +1,575 @@
+    "anima.ui.combat.ammo.damage.base.title": "Base des dégâts",
+    "anima.ui.combat.ammo.damage.title": "Dégâts",
+    "anima.ui.combat.ammo.integrity.base.title": "Base Solidité",
+    "anima.ui.combat.ammo.integrity.title": "Solidité",
+    "anima.ui.combat.ammo.name.title": "Nom",
+    "anima.ui.combat.ammo.presence.base.title": "Base présence",
+    "anima.ui.combat.ammo.presence.title": "Présence",
+    "anima.ui.combat.ammo.quality.title": "Qualité",
+    "anima.ui.combat.ammo.special.title": "Spécial",
+    "anima.ui.combat.ammo.title": "Munitions",
+    "anima.ui.combat.armors.at.cold.base.title": "FR",
+    "anima.ui.combat.armors.at.cold.title": "FR",
+    "anima.ui.combat.armors.at.cut.base.title": "TR",
+    "anima.ui.combat.armors.at.cut.title": "TR",
+    "anima.ui.combat.armors.at.electricity.base.title": "ELE",
+    "anima.ui.combat.armors.at.electricity.title": "ELE",
+    "anima.ui.combat.armors.at.energy.base.title": "ENE",
+    "anima.ui.combat.armors.at.energy.title": "ENE",
+    "anima.ui.combat.armors.at.heat.base.title": "CHA",
+    "anima.ui.combat.armors.at.heat.title": "CHA",
+    "anima.ui.combat.armors.at.impact.base.title": "CON",
+    "anima.ui.combat.armors.at.impact.title": "CON",
+    "anima.ui.combat.armors.at.none.title": "-",
+    "anima.ui.combat.armors.at.thrust.base.title": "PER",
+    "anima.ui.combat.armors.at.thrust.title": "PER",
+    "anima.ui.combat.armors.equipped.title": "Equipé?",
+    "anima.ui.combat.armors.integrity.base.title": "Base Solidité",
+    "anima.ui.combat.armors.integrity.title": "Solidité",
+    "anima.ui.combat.armors.isEnchanted.title": "Est-ce enchanté?",
+    "anima.ui.combat.armors.localization.breastplate.title": "Plastron",
+    "anima.ui.combat.armors.localization.complete.title": "Complète",
+    "anima.ui.combat.armors.localization.head.title": "Tête",
+    "anima.ui.combat.armors.localization.nightdress.title": "Chemise",
+    "anima.ui.combat.armors.localization.title": "Localisation",
+    "anima.ui.combat.armors.movementRestriction.base.title": "Base Restrict. Movem.",
+    "anima.ui.combat.armors.movementRestriction.final.title": "Restrict. Movem.",
+    "anima.ui.combat.armors.name.title": "Nom",
+    "anima.ui.combat.armors.naturalPenalty.base.title": "Base de Malus naturel",
+    "anima.ui.combat.armors.naturalPenalty.title": "Malus Naturel",
+    "anima.ui.combat.armors.presence.base.title": "Présence de Base",
+    "anima.ui.combat.armors.presence.title": "Présence",
+    "anima.ui.combat.armors.quality.title": "Qualité",
+    "anima.ui.combat.armors.title": "Armures",
+    "anima.ui.combat.armors.type.hard.title": "Dur",
+    "anima.ui.combat.armors.type.natural.title": "Naturel",
+    "anima.ui.combat.armors.type.soft.title": "Souple",
+    "anima.ui.combat.armors.type.title": "Type",
+    "anima.ui.combat.armors.wearArmorRequirement.base.title": "Base de Port d'Armure Req.",
+    "anima.ui.combat.armors.wearArmorRequirement.title": "Port d'Amure Req.",
```

Comment:
ther is a mistake Port d'Amure Req.

the right message is : Port d'Armure Req.

Reply:
Fixed.